fido-device-onboard.spec

```diff
@@ -103,8 +107,8 @@ Requires: openssl-libs >= 3.0.1-12
 %{_mandir}/man1/fdo-owner-onboarding-server.1*
 %{_libexecdir}/fdo/fdo-owner-onboarding-server
 %{_libexecdir}/fdo/fdo-serviceinfo-api-server
-%{_docdir}/fdo/owner-onboarding-server.yml
-%{_docdir}/fdo/serviceinfo-api-server.yml
+%{_docdir}/examples/owner-onboarding-server.yml
+%{_docdir}/examples/serviceinfo-api-server.yml
```

Comment:
but, yes I think we need this to actually be named %{_docdir}/%{name}/examples/ and same for the ones below.